src/app/utils/open-api-parser.ts

```diff
@@ -0,0 +1,50 @@
+export function parseOpenApiResponse(params: any) {
+  const { options: { paths }, url } = params;
```

Comment:
Looks like this function expects params to be a well-defined type. I think, instead of using `any`, we should define the type and optionally destructure in the parameter list. Example: https://gist.github.com/rileyhilliard/da10fe11b68d1b7feb31ad119c59d516#file-correct-destructure-ts

The advantage of this will be that if someone is to call this function, they will know in which format their data should be. In the current state, they can pass in an object of any form, even though the first line in the function is expecting a well formed input.